build/webpack.dev.js

```diff
@@ -0,0 +1,32 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.base.js').config;
+const webpack = require("webpack");
+const path = require('path');
+
+const config = merge(common, {
+    mode: 'development',
+    entry: {
+        'dash.all': './index.js',
+        'dash.mss': './src/mss/index.js'
```

Comment:
as well here: offline build is missing. This breaks sample page http://localhost:3000/samples/offline/index.html

Reply:
Done